2-copy-of-code/lesson-17/data/car.js

```diff
@@ -56,6 +56,31 @@ class Car {
   }
 }
 
+class RaceCar extends Car {
+  acceleration;
+
+  constructor(carDetails) {
+    super(carDetails);
+    this.acceleration = carDetails.acceleration;
+  }
+
+  go() {
+    this.speed += this.acceleration;
+
+    if (this.speed > 300) {
+      this.speed = 300;
+    }
+  }
+
+  openTrunk() {
+    console.log('Race cars do not have a trunk.');
+  }
+
+  closeTrunk() {
+    console.log('Race cars do not have a trunk.');
+  }
+}
```

Comment:
And what if I want to override displayInfo() in the  RaceCar ,  but the model and the brand are private?

Reply:
If the model and the brand are private then you will have to use Public Getters and Setters because you won't be able to directly access them.